diff -cr ED.r82/ED/src/driver/edmain.f90 ED.r83/ED/src/driver/edmain.f90
*** ED.r82/ED/src/driver/edmain.f90	2012-07-01 12:02:28.000000000 -0500
--- ED.r83/ED/src/driver/edmain.f90	2013-09-15 20:51:40.418160793 -0500
***************
*** 49,54 ****
--- 49,59 ----
  
    integer :: nslaves
  
+   ! isingle: force non mpi run
+   !    0 iff normal run with potential MPI
+   !    1 iff no MPI init is called
+   integer :: isingle=0
+ 
    ! scratch
  
    integer :: n
***************
*** 88,96 ****
    ! if MPI execution, machnum returns process rank and machsize process size;
  
    numarg=numarg+1
!   call MPI_Init(ierr)                                              
!   call MPI_Comm_rank(MPI_COMM_WORLD,machnum,ierr)                  
!   call MPI_Comm_size(MPI_COMM_WORLD,machsize,ierr)                 
    write (*,'(a)')       '+--- Parallel info: -------------------------------------+'
    write (*,'(a,1x,i6)') '+  - Machnum  =',machnum
    write (*,'(a,1x,i6)') '+  - Machsize =',machsize
--- 93,112 ----
    ! if MPI execution, machnum returns process rank and machsize process size;
  
    numarg=numarg+1
!   do n = 1, numarg
!       if (cargs(n)(1:2) == '-s') then
!          isingle=1
!       end if
!   end do
! 
!   if (isingle == 0) then
!      call MPI_Init(ierr)                                              
!      call MPI_Comm_rank(MPI_COMM_WORLD,machnum,ierr)                  
!      call MPI_Comm_size(MPI_COMM_WORLD,machsize,ierr)                 
!   else
!      machnum=0
!      machsize=1
!   end if
    write (*,'(a)')       '+--- Parallel info: -------------------------------------+'
    write (*,'(a,1x,i6)') '+  - Machnum  =',machnum
    write (*,'(a,1x,i6)') '+  - Machsize =',machsize

